11-recommendation/requirements.txt

```diff
@@ -0,0 +1,59 @@
+altair==5.5.0
```

Comment:
not that it matters too much, but do we actually need every lib here? like `GitPython` `h11` etc